kolibri/plugins/management/assets/src/vue/class-enroll-page/confirm-enrollment-modal.vue

```diff
@@ -94,6 +94,12 @@
 
 <style lang="stylus" scoped>
 
+  ul
+    list-style: none
+    margin: 20px 0
+    li
+      line-height: 1.8em
+
```

Comment:
Recommend making class names rather than using tag names. This can introduce bugs later if someone else adds a new list.

Also, best to [avoid nesting](http://thesassway.com/intermediate/avoid-nested-selectors-for-more-modular-css) `li` under `ul`